.ci/jobs/e2e-testing-integrations-daily-mbp.yml

```diff
@@ -9,7 +9,7 @@
     scm:
       - github:
           branch-discovery: no-pr
-          head-filter-regex: '(master|\d+\.\d+\.x|\d+\.x)'
+          head-filter-regex: '(master|7\.x|7\.11\.x|7\.10\.x)'
```

Comment:
Would this be better/simpler/more readable?
```suggestion
          head-filter-regex: '(master|7\.x|7\.1[01]\.x)'
```

Reply:
I'd say explicit regex might be easier to read in the future

Reply:
You mean the first one, or the condensed, second one?

Reply:
I prefer the first one, no the suggested change

Reply:
Thanks for the feedback! Merging now